config.php

```diff
@@ -10,9 +11,10 @@
     'class' => 'humhub\modules\gallery\Module',
     'namespace' => 'humhub\modules\gallery',
     'events' => [
-        ['class' => Menu::className(),'event' => Menu::EVENT_INIT, 'callback' => ['humhub\modules\gallery\Events', 'onSpaceMenuInit']],
-        ['class' => ProfileMenu::className(),'event' => ProfileMenu::EVENT_INIT, 'callback' => ['humhub\modules\gallery\Events','onProfileMenuInit']],
-        ['class' => Sidebar::className(),'event' =>  BaseMenu::EVENT_INIT, 'callback' => ['humhub\modules\gallery\Events','onSpaceSidebarInit']]
+        ['class' => Menu::class,'event' => Menu::EVENT_INIT, 'callback' => [Events::class, 'onSpaceMenuInit']],
+        ['class' => ProfileMenu::class,'event' => ProfileMenu::EVENT_INIT, 'callback' => [Events::class,'onProfileMenuInit']],
+//        ['class' => SpaceSidebar::class,'event' =>  BaseMenu::EVENT_INIT, 'callback' => [Events::class,'onSpaceSidebarInit']],
```

Comment:
Please remove such dead code

Reply:
All right! I will do it! I will remove the line number 16.

Thanks!